src/js/components/Volunteer/volunteer.scss

```diff
@@ -0,0 +1,9 @@
+.btn-volunteer-title{
+    border-radius: 100px !important;
```

Comment:
The usage of !important has been preached to your peers.
Do not make use of it unless it is your last option